Automattic/wp-calypso
# Performance Profiler: Add Migration Banner and Disclaimer

Fixes [Performance Profiler: Create disclaimer section for landing page and result page](https://github.com/Automattic/dotcom-forge/issues/8565)

## Proposed Changes

* Add migration banner
* Add logo bar
* Make all sections fit in `1056px`
* Update the `tested on` text to display real data

## Why are these changes being made?

To match the i2 version of Performance Profiler Project

## Testing Instructions

* Go to `/speed-test-url?url=wordpress.com`
* Check if the sections on the bottom of the page looks similar to the image below
* Check if the `Migrate your site` CTA leads you to a page with your domain filled

![CleanShot 2024-08-15 at 17 30 39@2x](https://github.com/user-attachments/assets/36f0dd32-344a-4b8a-9bca-884483ef5dad)


## Pre-merge Checklist
